### PR TITLE
Several README.rst improvements

### PR DESCRIPTION
**Pull request by Julian-O**
The first few commits are less controversial. The last few are more substantial.

### CODE REVIEW

**Comment by MatthieuDartiailh** [comment]
This looks really good ! Thanks.
Let me know when you consider that you are done and I will merge.

File: README.rst [hunk 1]
```diff
@@ -3,56 +3,121 @@ Welcome to Enaml
...
-  :target: http://enaml.readthedocs.io/en/latest/?badge=latest
-  :alt: Documentation Status
+Enaml is a programming language and framework for creating professional-quality
+user interfaces with minimal effort. 
 
+What you get
+===========================
+* A declarative programming language, with a Pythonic flavour.
+* Dozens of widgets, ready to go out-of-the-box (built on QT).
+* A constraints-based layout engine (built on Cassowary).
```
Comment (sccolbert) [inline README.rst]: s/Cassowary/Kiwi

**Comment by Julian-O** [comment]
Okay, that's all for now.

File: README.rst [hunk 2]
```diff
@@ -1,58 +1,127 @@
 Welcome to Enaml
 ================
-
+
```
Comment (MatthieuDartiailh) [inline README.rst]: Please remove the white spaces.

**Comment by MatthieuDartiailh** [comment]
Sorry I misread the changes for the image. 
Merging now and thanks for this, the README looks much better now !

**Pull request merged**